pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/golang/protobuf/proto"
 	"github.com/lf-edge/eve/api/go/attest"
 	zcert "github.com/lf-edge/eve/api/go/certs"
+	"github.com/lf-edge/eve/api/go/evecommon"
```

Comment:
Don't use the API definitions. Only zedagent should use them when doing protobuf messages.